sdk/translation/azure-ai-translation-document/azure/ai/translation/document/_client.py

```diff
@@ -118,6 +140,39 @@ def begin_translation(self, inputs, **kwargs):
                 :caption: Translate the documents in your storage container.
         """
 
+        continuation_token = kwargs.pop("continuation_token", None)
+
+        try:
+            inputs = kwargs.pop('inputs', None)
+            if not inputs:
+                inputs = args[0]
+            inputs = DocumentTranslationInput._to_generated_list(inputs) \
+                if not continuation_token else None  # pylint: disable=protected-access
+        except (AttributeError, TypeError, IndexError):
+            try:
+                source_url = kwargs.pop('source_url', None)
+                if not source_url:
+                    source_url = args[0]
+                target_url = kwargs.pop("target_url", None)
+                if not target_url:
+                    target_url = args[1]
+                target_language_code = kwargs.pop("target_language_code", None)
+                if not target_language_code:
+                    target_language_code = args[2]
+                inputs = [
+                    _BatchRequest(
+                        source=_SourceInput(
+                            source_url=source_url
+                        ),
+                        targets=[_TargetInput(
+                            target_url=target_url,
+                            language=target_language_code
+                        )]
+                    )
+                ]
+            except (AttributeError, TypeError, IndexError):
+                raise ValueError("Pass either 'inputs' or 'source_url', 'target_url', and 'target_language_code'")
```

Comment:
small nit: in the response you can say pass "inputs" for multiple inputs, or these for single